src/components/Button/Button.js

```diff
@@ -54,12 +55,12 @@ const Button = styled.button`
     color: ${get('colors.text.disabled')};
     border-color: ${get('colors.background.disabled')};
     cursor: default;
+    opacity: 0.5;
```

Comment:
Based on the original story it appears `hover` should be darker, however lowering the opacity would lighten on hover, here's the original spec: 

- Hover state does not use the 'primary dark' color, instead uses a slightly darker normal primary color.
- Disabled hover icons are not visible; they should be white.

Reply:
lower the opacity is the requirement from the story. I have already showed the changes to Shaun and he approved the look.